student-work/chana-willinger/Api-Final Project/POST.md

```diff
@@ -1 +1,68 @@
+POST/lunch
 
+This is a post order for lunch, to send to the kitchen. When an OK response is received, the order is printed in the kitchen so the cook can start cooking.
+
+**Example Response
+
+```curl 
+
+-H "Content-Type: application/json" -X POST -d'{
+
+```JSON
+
+   "mealType":"lunch",
+   "mealCat":{
+  	"main":"burgerMeal",
+  	"burger":{
+     	"pattyType":"beef",
+     	"pattyQty":1,
+     	"pattyWeightG":300,
+     	"pattyCook":"MR",
+     	"bunType":"wholeWheat",
+     	"condiment1":"ketchup",
+     	"condiment2":"secretSauce",
+     	"topping1":"lettuce",
+     	"topping2":"pickles",
+     	"topping3":"None",
+     	"topping4":"None",
+  	},
+  	"sides":{
+     	"side1":{
+        	"type":"frenchFries",
+        	"size":"large"
+     	},
+     	"side2":{
+        	"type":"none",
+        	"size":""
+     	}
+  	},
+  	"drink":{
+     	"type":"Coke",
+     	"size":"large",
+     	"ice":true
+  	}
+   }
+},
+http://URL/
+```
+**BurgerMeal Guide
+
```

Comment:
For the Table,  in the description column, please use capital letters at the beginning of sentences.